grails-app/domain/org/pih/warehouse/product/ProductPackage.groovy

```diff
@@ -39,7 +40,7 @@ class ProductPackage implements Comparable<ProductPackage>, Serializable {
     String description        // Description of the package
     String gtin                // Global trade identification number
     Integer quantity        // Number of units (each) in the box
-    Float price
+    ProductPrice productPrice
```

Comment:
Minor change. Move this down below the productSupplier as they are related. The quantity and uom should be together.

Reply:
@jmiranda should it stay as Float price instead of ProductPrice productPrice (according to your comment above: https://github.com/openboxes/openboxes/pull/2163#discussion_r567557329)

Reply:
It's up to you. I was saying the refactoring from price to productPrice was not a requirement for this ticket. But since you've already made the change, I'm ok with keeping it.

Reply:
Ok, then I'll keep it.